src/v0/destinations/customerio/transform.js

```diff
@@ -113,6 +113,7 @@ function processSingleMessage(message, destination) {
       logger.error(`could not determine type ${messageType}`);
       throw new InstrumentationError(`could not determine type ${messageType}`);
   }
+  evName = evName ? String(evName) : evName;
```

Comment:
Can we try to use validEventType from index.js which checks for event type to be string otherwise throws error ?

Reply:
@sanpj2292 @krishna2020 Should we throw error or convert it to string in this case?

Reply:
IMHO, if our docs say that eventName should be string, we should throw an error. @krishna2020 can you confirm on this ?

Reply:
@anantjain45823 here the check is on eventName, not on eventType. Checked your recent PR where you added this - https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/util/index.js#L2072

@ujjwal-ab Rename the above function name to `validateEventName` & re-use it. We are good to throw error in CustomerIO case as event name is being used for comparison in the code.

Reply:
Got it.

Reply:
Here, in customerio. We are mapping `evName` from page name and url also in case of page event.